src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,949 @@
+                  example: \"fr-CA fr en\" represents a preference for French as spoken in Canada, then French (without a region designation), followed by English (without a region designation).
+                id_token_hint:
+                  type : string
+                  description: ID Token previously issued by the Authorization Server being passed as a hint about the End-User's current or past authenticated session with the Client. If the End-User identified by the ID Token is logged in or is logged in by the request, then the Authorization Server returns a positive response.
+                login_hint:
+                  type : string
+                  description: Hint to the Authorization Server about the login identifier the End-User might use to log in (if necessary). 
+                acr_values:
+                  type : string
+                  description: Requested Authentication Context Class Reference values. Space-separated string that specifies the acr values that the Authorization Server is being requested to use for processing this Authentication Request, with the values appearing in order of preference.
+                amr_values:
+                  type : string
+                  description: AMR Values.
+                request:
+                  type : string
+                  description: This parameter enables OpenID Connect requests to be passed in a single, self-contained parameter and to be optionally signed and/or encrypted. The parameter value is a Request Object value. It represents the request as a JWT whose Claims are the request parameters.
+                request_uri:
+                  type : string
+                  description: This parameter enables OpenID Connect requests to be passed by reference, rather than by value. The request_uri value is a URL using the https scheme referencing a resource containing a Request Object value, which is a JWT containing the request parameters.
+                request_session_id:
+                  type : string
+                  description: Request session id.
+                session_id:
+                  type : string
+                  description: Session id of this call.
+                origin_headers:
+                  type : string
+                  description: Origin headers. Used in custom workflows.
+                code_challenge:
+                  type : string
+                  description: PKCE code challenge.
+                code_challenge_method:
+                  type : string
+                  description: PKCE code challenge method.
+                custom_response_headers:
+                  type : string
+                  description: Custom Response Headers.
+                claims:
+                  type : string
+                  description: Requested Claims.
+      responses:
```

Comment:
It is duplicate of error definition which we already have in line 180. Lets re-use code were possible instead of copy-paste. It's same endpoint with same errors (just different HTTP method GET vs POST)

Reply:
My bad, have rectified this.

Reply:
@pujavs swagger doc has example of re-using responses https://swagger.io/docs/specification/describing-responses/